tooling/nargo_fmt/tests/input/databus.nr

```diff
@@ -1,2 +1,2 @@
-fn main(x: pub u8, y: call_data u8) -> return_data u32 {}
+fn main(x: pub u8, y: call_data(0) u8) -> return_data u32 {}
```

Comment:
Can this be a new tests or do we not parse `y: call_data u8` any more?

Looking at this example from https://noir-lang.org/docs/noir/concepts/data_bus

Reply:
Ah, right, I forgot to mention about this. I had to change it because that's invalid syntax, this is the error I get:

```
error: Expected a ( but found u8
  ┌─ src/main.nr:1:33
  │
1 │ fn main(x: pub u8, y: call_data u8) -> return_data u32 {}
  │                                 --
```

I also couldn't find that syntax in the chumsky parser.

The test was passing because it failed to format the file, and the expected output is the same as the input... but because nothing changes, it passes.

Reply:
Or, put another way, it's currently invalid syntax. Maybe the intention was for it to be valid? In that case we could update the parser to support it (but maybe in a follow-up PR, given that the current parser doesn't handle it either).

Reply:
This should be invalid syntax, can you add some perturbation to the input file so that it needs to modify it in order to match the output?

Reply:
Done!